vcpkg/get_started/get-started-vs.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Install and use packages with vcpkg in Visual Studio
+description: Tutorial guides the user through the process of installing and using packages with vcpkg and Visual Studio.
+zone_pivot_group_filename: zone-pivot-groups.json
+zone_pivot_groups: operating-systems
+author: JavierMatosD
+ms.author: javiermat
+ms.topic: tutorial
+ms.date: 09/27/2023
+ms.prod: vcpkg
+#CustomerIntent: As a beginner C++ developer, I want to learn how to install and manage packages using vcpkg, so that I can easily set up and maintain C++ projects with necessary dependencies.
+---
+
+# Tutorial: Install and use packages with vcpkg
+
+This tutorial shows you how to create a C++ "Hello World" program that uses the `fmt` library with CMake, vcpkg and Visual Studio. You'll install dependencies, configure, build, and run a simple application.
+
+## Prerequisites
+
+- [Visual Studio with C++ development workload](https://visualstudio.microsoft.com/downloads/)
+- [CMake](https://cmake.org/download/)
+- [Git](https://git-scm.com/downloads)
+
+[!INCLUDE [setup-vcpkg](setup-vcpkg.md)]
+
+
+## 2 - Set up the Visual Studio project
+
+1. Create the project directory
+
+    We'll be working within a directory named `helloworld`. Ensure that this directory is a sibling of your vcpkg directory, meaning both directories should reside under the same parent directory.
+    
+    * Open Visual Studio
+    * Click on "Create a new project."
+    * Choose "Empty Project" as the template.
+    * Name your project "helloworld."
+    * Set the location to be the parent directory of your vcpkg installation.
+    * Check the box for "Place solution and project in the same directory."
+    * Click the "Create."
+  
+2. Generate a Manifest file
+
+    Create a vcpkg manifest file (`vcpkg.json`) by executing the following command in Visual Studio's built-in terminal:
+
+    ```console
+    ../vcpkg/vcpkg new --application
```

Comment:
I believe this will warn the user about conflicts between the in-box `VCPKG_ROOT` and their chosen direct path to the tool.